### PR TITLE
add some functions from openzeppelin-solidity test/helpers

### PR DESCRIPTION
**Pull request by mikec**
and bump version to 0.1.6

### CODE REVIEW

File: src/helpers/expectEvent.js [hunk 1]
```diff
@@ -1,5 +1,44 @@
-export default async function expectEvent (eventName, asyncFn) {
-  const { logs } = await asyncFn
-  const event = logs.find(e => e.event === eventName)
-  assert.isFalse(typeof event === 'undefined', 'Event must be triggered')
+import { BigNumber } from 'lk-web3-utils'
+import chai from 'chai'
+import chaiBigNumber from 'chai-bignumber'
+
+const should = chai.use(chaiBigNumber(BigNumber)).should()
```
Comment (jpkcambridge) [inline src/helpers/expectEvent.js:5]: What is this doing?

**Comment by jpkcambridge** [comment]
👍

File: src/helpers/shouldFail.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import chai from 'chai'
+
+const should = chai.should()
+
+async function shouldFailWithMessage (promise, message) {
```
Comment (ewilz) [inline src/helpers/shouldFail.js:5]: we never use `should` syntax in our tests. Why do you want to switch to should syntax?

**Comment by ewilz** [comment]
👍

**Review by ewilz — APPROVED** [review]

File: src/helpers/expectEvent.js (hunk 1 above)
Comment (mikec) [reply src/helpers/expectEvent.js:5]: getting the `should` module and configuring it to include BigNumber expectations

File: src/helpers/shouldFail.js (hunk 2 above)
Comment (mikec) [reply src/helpers/shouldFail.js:5]: just copying what zeppelin has

**Pull request merged**